recipes/ribocutter/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "ribocutter" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 1eaaaffd10411a7aadf61ef7009e0f73dde62017e6a70dd0751372e58d18203f
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - ribocutter=ribocutter.__main__:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - dnaio
+    - pandas
+    - pip
+    - python
+  run:
+    - dnaio
+    - pandas
+    - python
+
+test:
+  imports:
+    - ribocutter
+  commands:
+    - ribocutter --help
+
+about:
+  home: "https://github.com/Delayed-Gitification/ribocutter.git"
+  license: MIT
+  license_family: MIT
+  license_file: 
```

Comment:
Please include the license file.

Reply:
is it ok if I just removed this line? I have done so in the latest version